tests/policy-inspect-test.py

```diff
@@ -165,7 +165,7 @@ def test_undeclared_policy(self):
         script_dir = os.path.dirname(os.path.abspath(__file__))
         file_name = "result.json"
         input_file_name = os.path.join(script_dir,'data', file_name)
-        undeclared = UndeclaredComponent(filepath=input_file_name, format_type='json')
+        undeclared = UndeclaredComponent(filepath=input_file_name, format_type='json', style='sbom')
```

Comment:
now we are running unit tests automatically in a workflow, files will be automatically picked up if their name begins with `test_` so this should be renamed to `test_policy-inspect.py` or something like that